pkg/telemetry/phonehome/instance_config.go

```diff
@@ -0,0 +1,131 @@
+package phonehome
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/json"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/grpc/authn"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/set"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/pkg/version"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	apiPathsAnnotation = "rhacs.redhat.com/telemetry-apipaths"
+	csIDAnnotation     = "rhacs.redhat.com/cs-identity"
```

Comment:
I know we currently have some labels / annotations that follow the pattern `x.stackrox.io/y`, I'm not sure whether this should be the scheme we follow or not 🤷 

@SimonBaeumer do you have any particular guidance on the pattern we use for labels / annotations of our K8S objects?

Reply:
I took this `rhacs.redhat.com` from the fleetshard code.